packages/app-store/googlecalendar/lib/CalendarService.ts

```
@@ -286,6 +289,8 @@ export default class GoogleCalendarService implements Calendar {
              *  410 is when an event is already deleted on the Google cal before on cal.com
              *  404 is when the event is on a different calendar
              */
+            Sentry.captureException(err);
```

I think here we actually don't want to always report the error. Only if the error code is not equal to 404 and not equal to 410. Only when this code returns a rejected promise, should we report the error.